generators/createtodoapp/generator.mjs

```diff
@@ -146,7 +152,7 @@ ${chalk.cyan(`    mvn clean package -DskipTests`)}
 ${chalk.cyan(`    java -jar web\\target\\azure-spring-apps-todo-web-0.0.1-SNAPSHOT.jar`)}
 
 ${chalk.magentaBright(`Deploy on Azure Spring Apps with monthly free grants:`)}
-${chalk.cyan(`    https://review.learn.microsoft.com/en-us/azure/spring-apps/quickstart-for-web-app?branch=pr-en-us-227725`)}
+${chalk.cyan(`    https://learn.microsoft.com/azure/spring-apps/quickstart-deploy-web-app`)}
```

Comment:
I think we should use a aka.ms short link here

Reply:
https://shorturl.at/fjKU6

Reply:
https://aka.ms/asa/webapp-quickstart